tools/hermes_adapter/providers/gemini_cli_client.py

```
@@ -0,0 +1,319 @@
+        *,
+        timeout: int,
+        output_format: str = TEXT_FORMAT,
+    ) -> GeminiCliResult:
+        if output_format not in {TEXT_FORMAT, STREAM_JSON_FORMAT}:
+            raise ValueError(f"Unsupported Gemini CLI output format: {output_format}")
+
+        command = [str(self.cli_path), "-p", prompt]
+        if output_format == STREAM_JSON_FORMAT:
+            command.extend(["--output-format", STREAM_JSON_FORMAT])
+
+        result = subprocess.run(
```

Ruff (0.15.10)

[error] `subprocess` call: check for execution of untrusted input

(S603)